operations/research-and-development/product/developer-relations.md

```diff
@@ -34,15 +34,28 @@ Mattermosties are kind and considerate. We have a sense of humor and are general
 * **Be aware of cultural differences:** Occasionally there are cultural norms that change the course of conversation. Some examples include the reserved nature of many Scandinavian cultures or the brash, “we know this” nature of Americans. While no one expects you to obtain a minor in Cultural Anthropology or Sociology, be aware of certain norms from cultures outside your own.
 * **Assume everyone is an expert:** Regardless of any outward appearance, assume everyone we interact with has been working in their field and has top-level expertise. We do not assume anyone is junior or not tech-minded for any reason. They are approaching us with some level of interest. Let’s work to keep that interest kindled.
 
-## Definitions
+### Definitions
 
 * Event Engaged Attendee
   * **Virtual:** Any attendee who attends a talk or visits the booth interactively. This can include volunteering their information directly or having it provided by the organizers post-event.
   * **In-person:** Any attendee who attends a talk, offers their information at a booth or other station, interacts with any Mattermost representative with intent on further examining Mattermost. Interactions to simply look at Mattermost or to discuss philosophies of Mattermost may notbe considered "engaged" - this can be left to the respresentatives discretion.
 * Talk
   * Any keynote, speech, presentation, demo, or workshop given at an external event \(or at MatterCon\).
 
-## Assets
+### Assets
 
 * [Conference/Trip Report](https://docs.google.com/document/d/10AZQP5QaUHOd-cuiogA08TD6Im0XvS1qpOBIDcnJScI/edit?usp=sharing): To be used after speaking at an event or participating as a sponsor or in a sales or marketing capacity.
 
+### Process for inviting staff to events
+
+1. Allocate available passes to each department - e.g. 1 Eng, 1 Product, 1 Support, 2 Sales/CS, 1 Marketing. Allocation is typically based on
+  - Goal of an event --> e.g. if the main goal is to generate sales leads, we'd have a bigger sales/marketing presence. If the main goal is product or community awareness, then a bigger eng/product presence.
+  - Audience --> e.g. if the audience is mostly users (devs, etc), then we tended to bring more community, product or eng folks. Usually aligned with the goal of an event, but not always. Example is FOSDEM where we only bring the community team and engineers.
+2. Based on allocation, create a list of specific individuals we'd like to invite
+  - Prioritize those who have expressed interested in attending events.
+  - Prioritize those who are geographically close (lower costs, less time and effort for folks to attend).
+  - Prioritize specific needs, e.g. for KubeCon, we have previously invited engineers who worked on our Kubernetes operator and Cloud platform, instead of other engineers. For security events it has been our security engineers, etc
```

Comment:
```suggestion
  - Prioritize specific needs, e.g. for KubeCon, we have previously invited engineers who worked on our Kubernetes operator and Cloud platform, instead of other engineers. For security events it has been our security engineers, etc.
```